src/components/directory-content.jsx

```diff
@@ -293,93 +263,52 @@ const DirectoryContent = () => {
         event.stopPropagation();
     };
 
-    /* User interactions */
-    const contextualMixPolicies = useMemo(
-        () => ({
-            BIG: 'GoogleMicrosoft', // if !selectedUuids.has(selected.Uuid) deselects selectedUuids
-            ALL: 'All', // union of activeElement.Uuid and selectedUuids (currently implemented)
-        }),
-        []
-    );
-    const contextualMixPolicy = contextualMixPolicies.ALL;
```

Comment:
why did you remove this ?
also, I think the way it worked changed

before, when selecting several elements, when we right clicked another element, this element was added to the selection, it's not visually the case anymore (even if it seems to work)